tsl/test/expected/recompress_chunk_segmentwise.out

```diff
@@ -74,9 +74,9 @@ select _timescaledb_functions.recompress_chunk_segmentwise(:'chunk_to_compress_1
 
 -- check the ctid of the rows in the recompressed chunk to verify that we've written new data
 SELECT ctid, * FROM :compressed_chunk_name_1;
- ctid  |                                       time                                       | a | b | c | _ts_meta_count | _ts_meta_sequence_num |           _ts_meta_min_1            |           _ts_meta_max_1            
--------+----------------------------------------------------------------------------------+---+---+---+----------------+-----------------------+-------------------------------------+-------------------------------------
- (0,2) | BAAAApQ2Uhq14/////5TcU6AAAAAAwAAAAMAAAAAAAAO7gAFKG/+g/vGAAUob/+1KMUAAAADV+w1/w== | 2 |   | 2 |              3 |                    10 | Sun Jan 01 09:56:20.048355 2023 PST | Sun Jan 01 11:56:20.048355 2023 PST
+ ctid  | _ts_meta_count | _ts_meta_sequence_num | a | c |           _ts_meta_min_1            |           _ts_meta_max_1            |                                       time                                       | b 
```

Comment:
Is there a specific reason for not moving the segment_by columns (i.e., `a`, `c`) also to the end and have all `_ts_meta` columns first?

Reply:
I think filters on segmentby might be more frequent than ordering, so I put segmentby before the orderby metadata. In general for minmax columns, I think might be good to follow the order specified by the user, especially if we're going to introduce minmax for non-orderby columns. But this is just a vague intuition, haven't really done any research on it.